.github/workflows/prannouncer.yml

```diff
@@ -0,0 +1,14 @@
+name: "Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * MON-FRI"
+
+jobs:
+  prannouncer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: guardian/google-chats-pr-announcer@main
+        with:
+          google-webhook-url: ${{ secret.GOOGLE_WEBHOOK_URL }}
```

Comment:
Do we need to set the `https://github.com/guardian/google-chats-pr-announcer#github-repositories` and other required values from the action?

Reply:
Nope, they *should* default to the correct values by themselves.

The only reason you would want to set these yourselves is if you wanted to perhaps scan multiple repositories (which we might want to do) or if you're external to the guardian and you want to ignore different labels / bot users

Reply:
By default it just uses the name of the repository the action is created in, but we could set it to
`guardian/dotcom-rendering, guardian/frontend, guardian/discussion-rendering, ... etc` if we wanted to include the PR's from other repositories at the Guardian. I've just left it default for now, but we might want to do something different for the P&E/Dotcom space